server/index.js

```diff
@@ -48,14 +48,32 @@ async function setupApp() {
 
   });
 
+  app.post('/users', async (req, res) => {
+    try {
+      const { name, email, password, birthdate, notification_preference, biography, terms_agreed, newsletter_subscribed } = req.body;
```

Comment:
I think the frontend form fields have name attribute values using camelCaseLikeThis.

For example: `notificationPreference`

So we should extract the camelCase names.

Reply:
Ahh ok I don't know why I used the table names

Reply:
I didn't talk about it in bootcamp, but one good program to test POST requests is Postman.
https://www.postman.com/downloads/

You can skip registration, then change it to POST, then write the URL. Then you go to 

![image](https://github.com/nbktechworld/full-stack-web-dev/assets/13649002/50e30f27-4f8d-4b8d-a27d-89a70451294d)

Write the Body (raw, JSON).